src/zope/configuration/tests/test_config.py

```diff
@@ -79,31 +69,32 @@ class FauxPackage(object):
         c = self._makeOne()
         package = c.package = FauxPackage()
         package.__name__ = 'one.dot'
-        self.assertRaises(ConfigurationError, c.resolve, '....nonesuch')
+        with self.assertRaises(ConfigurationError):
+            c.resolve('....nonesuch')
 
     def test_resolve_bad_dotted_last_import(self):
         # Import error caused by a bad last component in the dotted name.
         from zope.configuration.exceptions import ConfigurationError
         c = self._makeOne()
-        exc = self.assertRaises(ConfigurationError,
-                          c.resolve, 'zope.configuration.tests.nosuch')
-        self.assertTrue('ImportError' in str(exc))
+        with self.assertRaises(ConfigurationError) as exc:
+            c.resolve('zope.configuration.tests.nosuch')
+        self.assertTrue('ImportError' in str(exc.exception))
```

Comment:
`self.assertIn('ImportError', str(exc.exception)` works on 2.7+ and gives a friendlier error. Since the line is changing anyway.